playground-projects/activity-playground/settings.gradle

```diff
@@ -20,8 +20,11 @@ pluginManagement {
 }
 plugins {
     id "playground"
```

Comment:
Why don't we make playground plugin apply the version, so we dont have to add this to every project?

Reply:
Yeah I'd prefer that but what's the magic incantation to make that work?